packages/flutter_tools/test/integration.shard/devtools_uri_test.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:flutter_tools/src/convert.dart';
+
+import '../src/common.dart';
+import 'test_data/basic_project.dart';
+import 'test_utils.dart';
+
+void main() {
+  late Directory tempDir;
+  final BasicProject project = BasicProject();
+
+  setUp(() async {
+    tempDir = createResolvedTempDirectorySync('run_test.');
+    await project.setUpIn(tempDir);
+  });
+
+  tearDown(() async {
+    tryToDelete(tempDir);
+  });
+
+  // Regression test for https://github.com/flutter/flutter/issues/126691
+  testWithoutContext('flutter run --start-paused prints DevTools URI', () async {
+    final Completer<void> completer = Completer<void>();
+    final RegExp matcher = RegExp(r'The Flutter DevTools debugger and profiler on');
```

Comment:
Can this just be a `String`?

Reply:
I think the test I used as a template for this one was actually matching the URI, so this can probably be `String` instead.